pgo/cases/joomla/TrainingCaseHandler.php

```diff
@@ -48,8 +48,6 @@ protected function getToolFn() : string
 
 	protected function setupDist() : void
 	{
-		$port = $this->getHttpPort();
-		$host = $this->getHttpHost();
```

Comment:
Please keep these two calls where ever in the `TrainingCaseHandler` classes. The `TrainingCaseHandler::init()` needs to setup the server config, these calls will generate appropriate config items if not present. Some cases like Wordpress use a cli tool which needs this info, too. Cases like Symfony don't need this however. Please check carefully to not to remove method calls as that might lead to bugs.

Thanks.

Reply:
Thank you, indeed, I did not notice. Can be rename these methods? Something like `getOrMakeHttpPort`/`getOrInitHttpPort`.

Reply:
Could rename, yep. Not critical. 

Thanks.